zhaquirks/tuya/tuya_thermostat.py

```diff
@@ -0,0 +1,451 @@
+"""Tuya TS0601 Thermostat."""
+
+import copy
+
+from zigpy.quirks.v2 import BinarySensorDeviceClass, EntityType
+from zigpy.quirks.v2.homeassistant import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfEnergy,
+    UnitOfPower,
+    UnitOfTemperature,
+)
+from zigpy.quirks.v2.homeassistant.sensor import SensorDeviceClass, SensorStateClass
+from zigpy.types import t
+from zigpy.zcl import foundation
+from zigpy.zcl.clusters.hvac import Thermostat
+
+from zhaquirks.tuya import TUYA_SET_TIME, TuyaTimePayload
+from zhaquirks.tuya.builder import TuyaQuirkBuilder
+from zhaquirks.tuya.mcu import TuyaAttributesCluster, TuyaMCUCluster
+
+
+class RegulatorPeriod(t.enum8):
+    """Tuya regulator period enum."""
+
+    FifteenMin = 0x00
+    ThirtyMin = 0x01
+    FortyFiveMin = 0x02
+    SixtyMin = 0x03
+    NinetyMin = 0x04
```

Comment:
I think other enums include the number like `_90_min` or `Min_90`? Do we want to do this here and for the enums below where numbers are spelled out at the moment?

In the future, we'll try to make enums translatable with https://github.com/zigpy/zha/pull/86.
These would act as the translation key then. Just fyi if this affects your decision.

Reply:
We can also merge them like this for now and streamline all enums in another PR, but we'd need to do that before the 2025.2.0 beta release date, so we don't mess up any actual HA entities.